Resources/Prototypes/Catalog/VendingMachines/Inventories/salvage.yml

```diff
@@ -9,6 +9,7 @@
     Floodlight: 2 # DeltaV - upstream removed them
     HandheldGPSBasic: 2
     RadioHandheld: 2
+    WeaponGrapplingGun: 2
```

Comment:
Following review, we'd appreciate this commented out. Grapple guns break the physics too much and may be pending removal from research also.